### DISCUSSION TITLE
useHead is depecrated

### DISCUSSION THREAD

**Discussion by 3aluw** [nuxt/nuxt#22728] [Questions]
Hello, I was trying to build a nuxt module so I imported useHead inside plugin.ts
`import {addRouteMiddleware, useHead,  useRuntimeConfig} from "#app"`
However, vscode crossed over useHead and propmted this : 
```
'useHead' is deprecated.ts(6385)
index.d.ts(6, 5): The declaration was marked as deprecated here.
```
useHead is working as intended, so what does this warning mean? Thank you

**Comment by WailanTirajoh** [comment]
false positive?

https://github.com/unjs/unhead/blob/main/packages/unhead/src/composables/useHead.ts

seems okay here. there's no @deprecated

**Comment by 3aluw** [comment]
Yeah, that's weird, I couldn't find @depecrated in the function declaration.
I'm using the nuxi module boilerplate.  

![Screenshot (2)](https://github.com/nuxt/nuxt/assets/82264545/a5b87882-db0a-41f7-a91d-7341837f8a36)
![Screenshot (3)](https://github.com/nuxt/nuxt/assets/82264545/48205614-9fff-4903-9731-35dad332a073)

**Reply by srad** [reply]
Yep, type still annotated/marked as deprecated  my the type-checker. Someone lost overview or it is actually deprecated

**Comment by Gwynerva** [comment]
@danielroe It is still marked as deprecated which can be confusing. Should I create an issue on this?

**Reply by danielroe** [reply]
It should not be imported from `#app`. Instead import it from `#imports`
